1_space_odyssey/index.html

```diff
@@ -18,7 +18,7 @@
 
 <body class="viewport-window__lunar-landing">
     <h1>2020: A Space Odyssey <span>lunar landing</span></h1>
-    <script src="./js/_adventures-9000/app.js"></script>
+    <script src="./js/trond-9000/app.js"></script>
```

Comment:
please revert this file, and push the change.

and then on todays task do not "commit" the index file to github